src/automation.ts

```diff
@@ -19,15 +19,26 @@ const init = async (workflows: Workflow[], options: Options) => {
   setConfig(options)
 }
 
-const buildEmailAction = (emails: EmailAction[], sendTo: string) => {
+const buildEmailAction = (emails: EmailAction[], sendTo: string, params?:BodyParam[]) => {
```

Comment:
Let's make this strict by removing the optional and setting a default value `params: BodyParam[] = []`

Reply:
Done 👍